### ISSUE TITLE
App doesn't start

### ISSUE THREAD

**Issue by ferferga** [UnicornsOnLSD/finamp#19]
After an update to 0.2.0 arm64 release I get this following screen:

![Screenshot_20210411_004342_com unicornsonlsd finamp](https://user-images.githubusercontent.com/10274099/114286448-1f24ad00-9a5f-11eb-9ad7-7fb1e55242af.jpg)

After clearing data everything works fine :).

**Comment by UnicornsOnLSD** [comment]
I got that when I updated too, but I don't know why. The only thing that could be "typeId 61" would be this line, which hasn't been changed in months: https://github.com/UnicornsOnLSD/finamp/blob/42813bdf4da0b743cb79582dee221942a75769b3/lib/models/JellyfinModels.dart#L1170

It could be to do with me removing the old logging system. If the old log DB exists, 0.2 opens it and immediately deletes it. It could be getting confused since the HiveType no longer exists, and loops back around to the first HiveType, which is only HiveType with typeId 61.

I'll add back those old HiveTypes in the next release just in case.
